cli/cli.py

```diff
@@ -0,0 +1,297 @@
+import json
+import configparser
+import hashlib
+import os
+import logging
+from pathlib import Path
+from OpenSSL import crypto
+from socket import gethostname
+
+CONF_FILE = '../../conf/lxdui.conf'  # lxdui.conf.dir
+APP_NAME = 'LXDUI'  # lxdui.app.alias
+LOG_DIR = '/Users/vetoni/PycharmProjects/lui-cli/logs'
+
```

Comment:
We will have to move CONF_FILE, APP_NAME and LOG_DIR to a separate configuration file.
Let's agree on this on today's meeting, we have similar scenarios.

LOG_DIR = '/Users/vetoni/PycharmProjects/lui-cli/logs
We have to be careful with absolute paths.